src/optype.rs

```diff
@@ -1,594 +1,432 @@
-     * Fixed instance of a \ref OpType::ISWAP gate with parameter \f$ 1.0 \f$:
-     * \f$ \left[ \begin{array}{cccc} 1 & 0 & 0 & 0 \\ 0 & 0 & i & 0 \\ 0 & i
-     * & 0 & 0 \\ 0 & 0 & 0 & 1 \end{array} \right] \f$
-     */
+    /// Fixed instance of a \ref OpType::ISWAP gate with parameter \f$ 1.0 \f$:
+    /// \f$ \left[ \begin{array}{cccc} 1 & 0 & 0 & 0 \\ 0 & 0 & i & 0 \\ 0 & i
+    /// & 0 & 0 \\ 0 & 0 & 0 & 1 \end{array} \right] \f$
     ISWAPMax,
 
+    /// ISwap gate with extra `Rz`s on each qubit
+    // TODO: Matrix description
     PhasedISWAP,
 
-    /**
-     * Multiply-controlled \ref OpType::Ry
-     *
-     * The phase parameter is defined modulo \f$ 4 \f$.
-     */
+    /// Multiply-controlled \ref OpType::Ry
+    ///
+    /// The phase parameter is defined modulo \f$ 4 \f$.
     CnRy,
 
-    /**
-     * Multiply-controlled \ref OpType::X
-     */
+    /// Multiply-controlled \ref OpType::X
     CnX,
 
-    /**
-     * See \ref CircBox
-     */
+    /// See \ref CircBox
     CircBox,
 
-    /**
-     * See \ref Unitary1qBox
-     */
+    /// See \ref Unitary1qBox
     Unitary1qBox,
 
-    /**
-     * See \ref Unitary2qBox
-     */
+    /// See \ref Unitary2qBox
     Unitary2qBox,
 
-    /**
-     * See \ref Unitary3qBox
-     */
+    /// See \ref Unitary3qBox
     Unitary3qBox,
 
-    /**
-     * See \ref ExpBox
-     */
+    /// See \ref ExpBox
     ExpBox,
 
-    /**
-     * See \ref PauliExpBox
-     */
+    /// See \ref PauliExpBox
     PauliExpBox,
 
-    /**
-     * NYI
-     */
+    /// NYI
     CliffBox,
 
-    /**
-     * See \ref CustomGate
-     */
+    /// See \ref CustomGate
     CustomGate,
 
-    /**
-     * See \ref PhasePolyBox
-     */
+    /// See \ref PhasePolyBox
     PhasePolyBox,
 
-    /**
-     * See \ref QControlBox
-     */
+    /// See \ref QControlBox
     QControlBox,
 
-    /**
-     * See \ref ClassicalExpBox
-     */
+    /// See \ref ClassicalExpBox
     ClassicalExpBox,
 
-    /**
-     * See \ref Conditional
-     */
+    /// See \ref Conditional
     Conditional,
 
-    /**
-     * See \ref ProjectorAssertionBox
-     */
+    /// See \ref ProjectorAssertionBox
     ProjectorAssertionBox,
 
-    /**
-     * See \ref StabiliserAssertionBox
-     */
+    /// See \ref StabiliserAssertionBox
     StabiliserAssertionBox,
 
-    /**
-     * See \ref UnitaryTableauBox
-     */
+    /// See \ref UnitaryTableauBox
     UnitaryTableauBox,
 
+    ///
     #[cfg(feature = "tket2ops")]
     AngleAdd,
+    ///
     #[cfg(feature = "tket2ops")]
     AngleMul,
+    ///
     #[cfg(feature = "tket2ops")]
     AngleNeg,
+    ///
     #[cfg(feature = "tket2ops")]
     QuatMul,
+    ///
     #[cfg(feature = "tket2ops")]
     RxF64,
+    ///
     #[cfg(feature = "tket2ops")]
     RzF64,
+    ///
     #[cfg(feature = "tket2ops")]
     Rotation,
+    ///
     #[cfg(feature = "tket2ops")]
     ToRotation,
+    ///
     #[cfg(feature = "tket2ops")]
     Copy,
+    ///
     #[cfg(feature = "tket2ops")]
     Const,
```

Comment:
```suggestion

```

Reply:
I put this in a different PR, but may as well do it here.